eq-author/src/App/questionPage/Design/QuestionPageEditor/AnswersEditor/AnswerEditor/index.js

```diff
@@ -109,20 +127,52 @@ class AnswerEditor extends React.Component {
   render() {
     return (
       <Answer>
-        <AnswerType>{this.props.answer.type}</AnswerType>
+        <AnswerHeader data-test="answer-type">
+          <AnswerType>{this.props.answer.type}</AnswerType>
+
+          <Buttons>
+            <Tooltip
+              content="Move answer up"
+              place="top"
+              offset={{ top: 0, bottom: 10 }}
+            >
+              <MoveButton
+                disabled={!this.props.canMoveUp}
```

Comment:
As discussed, change this to `aria-disabled` so the button retains focus when "disabled".